src/jdk.jfr/share/classes/jdk/jfr/internal/PlatformRecording.java

```diff
@@ -176,7 +176,9 @@ public boolean stop(String reason) {
                 notifyIfStateChanged(newState, oldState);
                 close(); // remove if copied out
             } catch(IOException e) {
-                // throw e; // BUG8925030
+                Logger.log(LogTag.JFR, LogLevel.ERROR,
+                           "Unable to complete I/O operation when dumping recording \"" + getName() + "\" (" + getId() + ") due to: "
+                           + e.getMessage());
```

Comment:
I think you can remove e.getMessage() part. It will not provide useful information in case of a problem, due to how tings are now implemented. Even if it would be changed, I think it is good to avoid printing  file paths when using the default log level.

Reply:
removed.